src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -296,6 +330,20 @@ public void validateUserCanRead(List<String> sourcePaths, AuthenticatedUserReque
     }
   }
 
+  private void addPetServiceAccountToDatasetProject(
+      String projectId, String petServiceAccountEmail) {
+    logger.info("Adding pet service account permissions to dataset project");
```

Comment:
Would it be helpful to add more details to this log message, like the pet service account and the google project?